### ISSUE TITLE
Command line giving error

### ISSUE THREAD

**Issue by gitUser2k17** [ibm-datapower/datapower-configuration-manager#37]
Hi,

I have setup DCM and run the quickStart succesfully in ANT commandline. I have started with easyStart and created a domain but when I try to deploy a service it is giving me an error, I tried both .zip and .xcfg its the same error that I am getting. 

Can someone please help me ? Below are the details.

Command  

ant -f easyStart.ant.xml -Ddevice=dp01 -Denvironment=dev -Dservice=MPGSample import-from-def

I have defined all properties. I tried the same command with domain create and save as given in Wiki, no luck. So I simplified as above this doesn't work still.

Error

raw-policy.xml producing C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart\tmp/2_included_policies.xml
    [nxslt] Warning:  The encoding 'UTF-8' is not supported by the Java runtime.
    [nxslt] Warning: encoding "UTF-8" not supported, using UTF-8
    [nxslt] nxslt running C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager/src/expand-black-white-modify.xsl on C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart\tmp/2_included_policies.xml producing C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart\tmp/3_expanded-deployment-policies.xml
    [nxslt] Warning:  The encoding 'UTF-8' is not supported by the Java runtime.
    [nxslt] Warning: encoding "UTF-8" not supported, using UTF-8
    [nxslt] nxslt running C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager/src/write-deployment-policy.xsl on C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart\tmp/3_expanded-deployment-policies.xml producing C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart\tmp/4_gathered-deployment-policies.xml
    [nxslt] Warning:  The encoding 'UTF-8' is not supported by the Java runtime.
    [nxslt] Warning: encoding "UTF-8" not supported, using UTF-8
     [copy] Copying 1 file to C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart\tmp
     [echo] Importing C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart/MPGSample/MPGSample.xcfg into domain dp01 on x.x.x.x.
     [echo] ### ${import_success}=${import_success}
     [echo] ### ${import_errors}=${import_errors}
     [echo] Raw response to importing C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart/MPGSample/MPGSample.xcfg
     [echo] ${import_response} ${import_errors}
BUILD FAILED
C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\deploy.ant.xml:935: The following error occurred while executing this line:
C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\src\dcm-taskdefs.ant.xml:1654: Failed to import C:\Users\usr_2\Documents\svn_workspace\Automation\Datapower Configuration Manager\easystart/mpgSample/MPGSample.xcfg into domain dp01 on x.x.x.x

**Comment by richgroot** [comment]
It would be useful to see what is being returned from DataPower, so add
this to your command line:

-Dcapturesoma=soma.txt

This will capture all the requests and responses to/from DataPower.  The
soma.txt file will likely be large (> 1MB) since we're debugging an import.

You can send the file directly to me (richgroot@gmail.com) rather than
posting it on github.

Thanks,
Rich

**Comment by gitUser2k17** [comment]
Hi,

Thanks for the response. agree that would really be useful but unfortunately nothing is coming back. If you observe this line, it is printed as it is, I couldn't identify how these variables get populated with some content.

[echo] ${import_response} ${import_errors}

Thanks.

**Comment by richgroot** [comment]
Did you add -Dcapturesoma=soma.txt to the command line?  Was a file
produced named soma.txt?

**Comment by gitUser2k17** [comment]
Hi,

I have tried with some help from Rich and determined the root cause. I have a port number '5555' for XML management which is different from default but somewhere deep it was considering the default '5550' and trying to send a request to DP which obviously fails.  I have not seen where and why is it considering but for the time being, I reinstalled DP on my docker machine with port '5550' and it worked fine.
